<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="pselectLayout">
        <item>
         <widget class="QPushButton" name="pushButton_upload">
          <property name="text">
           <string>Upload new samples</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_left">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_project">
          <property name="text">
           <string>Project</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_project"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_mid">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_value">
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_value">
          <item>
           <property name="text">
            <string>k-factor (mean)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>k-factor (mean deviation)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hysterese (mean)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hysterese (mean deviation)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_right">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_top" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_detail">
          <property name="text">
           <string>Details</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_bot" native="true"/>
      </item>
      <item>
       <layout class="QGridLayout" name="detailselectLayout">
        <item row="0" column="9">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>B-Parameter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Sample</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Print char.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Design</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QScrollArea" name="scrollArea_sample">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_sample">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>83</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5"/>
          </widget>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Cycles</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>F-Parameter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Orientation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>A-Parameter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>G-Parameter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Material</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea_design">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_design">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>82</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_design"/>
          </widget>
         </widget>
        </item>
        <item row="0" column="11">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Steps</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QScrollArea" name="scrollArea_material">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_material">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>99</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QScrollArea" name="scrollArea_print">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_print">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QScrollArea" name="scrollArea_orientation">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_orientation">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>99</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QScrollArea" name="scrollArea_A">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_A">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>99</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QScrollArea" name="scrollArea_B">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_B">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="7">
         <widget class="QScrollArea" name="scrollArea_F">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_F">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>99</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QScrollArea" name="scrollArea_G">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_G">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QScrollArea" name="scrollArea_speed">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_speed">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>99</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QScrollArea" name="scrollArea_cycles">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_cycles">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13"/>
          </widget>
         </widget>
        </item>
        <item row="1" column="11">
         <widget class="QScrollArea" name="scrollArea_steps">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_steps">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>99</width>
             <height>734</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_14"/>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
